sw/device/tests/plic_all_irqs_test.h

```diff
@@ -0,0 +1,85 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_PLIC_ALL_IRQS_TEST_H_
+#define OPENTITAN_SW_DEVICE_TESTS_PLIC_ALL_IRQS_TEST_H_
+
+/**
+ * Helper macros for the auto-generated plic_all_irqs_test. These are not meant
```

Comment:
Maybe the file should have a name that reflects that? The file name seems to imply that it's an API, rather than being a helper. (E.g., `plic_all_irqs_test_helpers.h`.)

Reply:
Done.